src/pages/login.js

```diff
@@ -29,6 +31,9 @@ const IndexPage = () => {
       .finally(() => {
         setLoading(false)
       })
+      .catch(e => {
+        setIsInvalidLogin(true)
+      })
```

Comment:
Tiny thing. I think we should change to have the `finally`-block as the last part of the promise chain.

Reply:
Yep, no problem, done